scorecard/profile_data/indicators/current_debtors_collection_rate.py

```diff
@@ -36,7 +36,7 @@ class CurrentDebtorsCollectionRate(SeriesIndicator):
             {
                 "cube": "incexp",
                 "item_codes": [
-                    "0200", "0400", "1000",
+                    "0200", "0400", "1000 less item code 2000",
```

Comment:
mmm not sure this is gonna work. Then it will try and highlight item code "1000 less item code 2000"

I think we'll have to break this one out into its own whole object. It will be pretty verbose but at least technically correct and not another very special case in the link generation logic

Reply:
I could separate the items to look like this 0200, 0400, 1000, less item code, 2000

Reply:
sneaky. alright